src/util/util.cpp

```diff
@@ -33,38 +31,38 @@ void Util::update(QtCharts::QAbstractSeries* series, const QVector<double>& poin
         return;
     }
 
-    if(initPos > finalPos) {
+    if (initPos > finalPos) {
         qCDebug(util) << "InitPos need to be lower than finalPos.";
         return;
     }
 
     // Points per Meter
-    const float distPoints = numberOfPoints/(maxPoint - minPoint);
+    const float distPoints = numberOfPoints / (maxPoint - minPoint);
 
-    QXYSeries *xySeries = static_cast<QXYSeries *>(series);
+    QXYSeries* xySeries = static_cast<QXYSeries*>(series);
 
     // Use replace instead of clear + append, it's optimized for performance
     QVector<QPointF> realPoints;
     realPoints.reserve(numberOfPoints);
 
     // Start
-    const int lastStartPoint = int(distPoints*(initPos-minPoint));
-    #pragma omp for
-    for(int i = 0; i < lastStartPoint; i++) {
+    const int lastStartPoint = int(distPoints * (initPos - minPoint));
+#pragma omp for
+    for (int i = 0; i < lastStartPoint; i++) {
         realPoints << QPointF(i, 0);
     }
 
     // Data
-    const int lastDataPoint = int((finalPos - initPos)*distPoints);
-    const float dataIndexScale = points.length()/((finalPos - initPos)*distPoints);
-    #pragma omp for
-    for(int i = 0; i < lastDataPoint; i++) {
-        realPoints << QPointF(i + lastStartPoint, multiplier * points[static_cast<int>(i*dataIndexScale)]);
+    const int lastDataPoint = int((finalPos - initPos) * distPoints);
+    const float dataIndexScale = points.length() / ((finalPos - initPos) * distPoints);
```

Comment:
want to change the math rules?

Reply:
I was unable to find any math format in clang for that, I believe that will need to drop that if we are aiming to use clang-format.
@tcanabrava, any tips ?